src/olympia/signing/views.py

```diff
@@ -113,6 +114,39 @@ def check_throttles(self, request):
             return
         super().check_throttles(request)
 
+    # When DRF 3.12 is released, we can remove the custom check_permissions()
+    # and permission_denied() as it will contain the fix for
+    # https://github.com/encode/django-rest-framework/issues/7038
```

Comment:
Should we have an issue in addons-server for that, too?

Reply:
https://github.com/mozilla/addons-server/issues/14178